### ISSUE TITLE
Is my trained loss value reasonable?

### ISSUE THREAD

**Issue by deyingk** [dahiyaaneesh/peclr#7]
Hi, 

Could you please help me confirm my trained unsupervised loss function is in a reasonable range?
(Since the supervised training code is not released, I could't verify it on the FreiHand.)

The following is the loss function from the stage of unsupervised training. The backbone is ResNet152. After 100 epochs, the loss is around 3.60.

![Screenshot from 2022-01-20 16-26-23](https://user-images.githubusercontent.com/23550136/150443582-dc1f2021-af73-4b8d-906a-1df3445736e0.png)

Thanks so much!

**Comment by spurra** [comment]
Yes this looks reasonable.
